internal/controller/pdbwatcher_controller.go

```diff
@@ -197,7 +197,7 @@ func (r *PDBWatcherReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		WithEventFilter(predicate.Funcs{
 			// ignore status updates as we make those.
 			UpdateFunc: func(ue event.UpdateEvent) bool {
-				return ue.ObjectOld.GetGeneration() != ue.ObjectNew.GetGeneration()
+				return ue.ObjectOld.Spec.LastEviction != ue.ObjectNew.Spec.LastEviction
```

Comment:
don't think we want this. If you change the target we need to regenerate. is ther anything in spec we don't want to re-recocile on?